my_code.py

```diff
@@ -0,0 +1,33 @@
+import numpy as np
```

Comment:
It will be  a good idea to use: 
`#!/usr/bin/env python`
`# -*- coding: utf-8 -*-`